web-platform/website/src/components/pages/login/ValuePropView.tsx

```diff
@@ -0,0 +1,102 @@
+import { Col, Container, Row } from "reactstrap";
+import { style } from "typestyle";
+
+import { Strings } from "../../../resources/Strings";
+import { Colors, Spacing } from "../../../styles";
+import { FontSizeRaw, Fonts } from "../../../styles/Fonts";
+
+const strings = Strings.components.pages.login.getExtensionView;
+const loginStrings = Strings.components.pages.login.loginPageContentV2;
```

Comment:
We should create a new string category under components.pages.login.valueProp and move these strings here. That way ownership is clear and we can safely remove these strings if we ever remove this class.